htdocs/comm/propal/card.php

```diff
@@ -2008,7 +2008,8 @@
 			$sday = date("d", $tmpdte);
 			print $form->selectDate($syear."-".$smonth."-".$sday, 'date_livraison', '', '', '', "addprop");
 		} else {
-			print $form->selectDate(-1, 'date_livraison', '', '', '', "addprop", 1, 1);
+			$tmp_date_delivery = GETPOST('date_delivery') ? : -1;
```

Comment:
Getpost key name is 'date_delivery'
But name of field 'date_livraison'
Should not be the same ?

Reply:
I would say that everything should be English and whenever possible. 
But if you insist on changing this I can

Reply:
I think too that we must migrate step by step to english. But whatever is language, the name of variable must be consistent in all page.
So if the name of param is xxx for assignement, you should use xxx to read it. But here, there is 2 different name. Is it done on purpose ?